lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2352,6 +2401,99 @@ class ConvertAtenPermuteOp : public OpConversionPattern<AtenPermuteOp> {
 };
 } // namespace
 
+
+namespace {
+struct ConvertAtenSliceLikeOp : ConversionPattern {
+  ConvertAtenSliceLikeOp(TypeConverter &typeConverter, MLIRContext *context)
+      : ConversionPattern(typeConverter, MatchAnyOpTypeTag(), /*benefit=*/1,
+                          context) {}
+  LogicalResult
+  matchAndRewrite(Operation *op, ArrayRef<Value> operands,
+                  ConversionPatternRewriter &rewriter) const override {
+    if (!isa<AtenSliceTensorOp, AtenSelectIntOp>(op))
```

Comment:
We could transform `AtenSelectIntOp` to `AtenSliceTensorOp ` in https://github.com/llvm/torch-mlir/blob/main/lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp. AtenSelectIntOp(dim, index)=> AtenSliceTensorOp(dim, start=index, end=index+1,step=1)